tapir/accounts/views.py

```diff
@@ -45,8 +45,12 @@ def form_valid(self, form):
         with transaction.atomic():
             response = super().form_valid(form)
 
+            # remove password
             new_frozen = freeze_for_log(form.instance)
-            if self.old_object_frozen != new_frozen:
+            new_frozen.pop("password")
```

Comment:
Maybe we could use the `excluded_fields` thing like in tapir/log/models.py:203 ?
Something like 
```python
if hasattr(form.instance, "excluded_fields"):
    for field in form.instance.excluded_fields:
        old_frozen.pop(field)
        new_frozen.pop(field)
```

Reply:
You mean like
```python
class TapirUserUpdateView(
    PermissionRequiredMixin, UpdateViewLogMixin, TapirFormMixin, generic.UpdateView
):
    permission_required = PERMISSION_ACCOUNTS_MANAGE
    model = TapirUser
    form_class = TapirUserForm
    excluded_fields = ["password"]

    def form_valid(self, form):
        with transaction.atomic():
            response = super().form_valid(form)

            # remove password
            new_frozen = freeze_for_log(form.instance)
            old_frozen = self.old_object_frozen
            if hasattr(self, "excluded_fields"):
                for field in self.excluded_fields:
                    old_frozen.pop(field)
                    new_frozen.pop(field)
```
because I wasn't able to implement it as part of TapirUserForm...

Reply:
I was hopping we could put the `if hasattr(): ...` inside `freeze_for_log`, can you try that?
Then we have it for all the logs, as long as they have an `excluded_fields`.